service-base/src/main/java/org/eclipse/hono/service/resourcelimits/PrometheusBasedResourceLimitChecks.java

```diff
@@ -123,20 +131,56 @@ String value() {
      * @param cacheProvider The cache provider to use for creating caches for metrics data retrieved
      *                      from the prometheus backend or {@code null} if they should not be cached.
      * @throws NullPointerException if any of the parameters except cacheProvider are {@code null}.
+     * @deprecated Use {@link #PrometheusBasedResourceLimitChecks(WebClient, PrometheusBasedResourceLimitChecksConfig, 
+     *                  CacheProvider, Tracer)} instead.
      */
+    @Deprecated
```

Comment:
I do not really see why this constructor should be deprecated. We do not have any problem if anybody still uses it and doesn't have any need to trace execution, do we?
Also, I think the implementation could be
```
this(webClient, config, cacheProvider, NoopTracerFactory.create());
```
instead of duplicating the code. WDYT?

Reply:
Sounds good. I have pushed the changes, pls have a look.